packages/e2e-test-utils/package.json

```diff
@@ -27,7 +27,7 @@
   "main": "./src/index.js",
   "dependencies": {
     "@wordpress/url": "^3.48.0",
-    "expect-puppeteer": "^9.0.1"
+    "expect-puppeteer": "^9.0.2"
```

Comment:
The latest version is 10.0.0

Reply:
This was added automatically after running `npm update expect-puppeteer`. Should I update the version manually?

Reply:
Yes in that case manually would be fastest.  `npm update` also looks at dependencies and their requirements etc. 

Since Puppeteer updates involve a few packages, it's usually easiest to manually update the version numbers in `package.json` and then run `npm i`